tsconfig.json

```diff
@@ -1,9 +1,3 @@
 {
-  "compilerOptions": {
-    "target": "es2019",
-    "module": "commonjs",
-    "esModuleInterop": true,
-    "forceConsistentCasingInFileNames": true,
-    "strict": true
-  }
-}
+  "extends": "@tsconfig/node20/tsconfig.json"
+}
```

Comment:
While those changes ensure we have a compatible config with associated `node` version and reduce noise, those settings as overridden internally `esbuild` through the usage `platform`, `target` and `format` settings.